connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java

```diff
@@ -0,0 +1,92 @@
+package com.redhat.cloud.notifications.connector.email.config;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class EmailConnectorConfig {
```

Comment:
Could you please make it extend `ConnectorConfig` and add a method similar to this?
https://github.com/RedHatInsights/notifications-backend/blob/43ddfb0ff907df3f4e725184cf4f76aa016b0b44/connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowConnectorConfig.java#L21-L28

Reply:
Done! And added the parameters in the connector's ClowdApp file.

Reply:
Thanks! Now if you could log all config values, that would be even better :smile: This is used at connector startup to log all the connector config.

Reply:
Sure! I've logged all the additional properties. I couldn't use the `Map.of` statement because it was telling me that no constructor existed for that and what not.